config/initializers/active_job.rb

```diff
@@ -1,5 +1,11 @@
 ActiveJob::Base.queue_adapter = if Rails.env.test?
                                   :test
+                                elsif Rails.env.development? || Figaro.env.pt_mode == 'on'
+                                  # letter_opener_web (used during development only)
+                                  # doesn't currently work with sidekiq running in a separate
+                                  # container since it relies on a shared ./tmp directory.
+                                  # https://github.com/fgrehm/letter_opener_web/issues/35
+                                  :inline
                                 else
```

Comment:
I think thats a great idea.  Mailcatcher looks great since then the dev environment is even closer to prod since SMTP code is actually exercised.    Do you think I should do that in this PR or a followup?   I'm tempted to get this landed and iterate.    On that note would be able to verify if this PR works on mac?  I only have linux.